test/fixtures/e2e/bento/bento-timeago.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Bento Timeago</title>
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.ampproject.org/v0/bento-timeago-1.0.css"
```

Comment:
For my own curiosity: Is this CSS file available in the e2e test runner? It will have to be removed if not, but likely can be omitted even if it is available -- it only contains pre-upgrade styles.